scripts/generate_records/generate_db_records.js

```diff
@@ -0,0 +1,586 @@
+// @ts-check
+/* eslint-disable node/no-extraneous-require */
+/* eslint-disable no-await-in-loop */
+const pMap = require('p-map');
+const minimist = require('minimist');
+const cryptoRandomString = require('crypto-random-string');
+const fs = require('fs-extra');
+const Logger = require('@cumulus/logger');
+const {
+  GranulePgModel,
+  CollectionPgModel,
+  ProviderPgModel,
+  ExecutionPgModel,
+  GranulesExecutionsPgModel,
```

Comment:
maybe? I've skipped the things that didn't occur to me as "need to uploading bulk for a test" I'd like to stick to scope, but if you feel like that belongs in scope fair enough

Reply:
I would look into LP's data first. They definitely don't have any PDRs, so that can be added later. Rules, I'm not sure how much there are or if it matters for our testing for ES.

Reply:
I'd like to look at scoping this out, LP has no pdfs in the snap database, and rules only a handful

Reply:
after our conversation, I'm going to call this out of scope